tests/NodeMCU_Test_Environment.rst

```diff
@@ -0,0 +1,157 @@
+###########################
```

Comment:
Only just noticed...this is the only `.rst` doc we have. For consistency reasons we should probably convert this to markdown?

Reply:
I loathe Markdown, but if someone commits the conversion I won't be too upset.